routing/car_router.cpp

```diff
@@ -569,4 +490,84 @@ IRouter::ResultCode OsrmRouter::FindPhantomNodes(m2::PointD const & point,
   getter.MakeResult(res, maxCount);
   return NoError;
 }
+
+bool CarRouter::IsEdgeIndexExisting(Index::MwmId const & mwmId)
+{
+  MwmSet::MwmHandle const handle = m_pIndex->GetMwmHandleById(mwmId);
+  if (!handle.IsAlive())
+  {
+    ASSERT(false, ("m_mwmHandle is not alive."));
+    return false;
+  }
+
+  MwmValue const * value = handle.GetValue<MwmValue>();
+  ASSERT(value, ());
+  if (value->GetHeader().GetFormat() < version::Format::v8)
```

Comment:
> There was no section with this name.

Now we're thinking about a new name for the section. edgeidx does not reflect the content of the section. It'll be contain all the routing graph. So the section name will be probably change. So at first let's come to decision about section name. Please see discussion in the beginning of the PR.

> No need in this optimization.
I agree.